### PR TITLE
Fixed typos and made a few suggestions

### PR DESCRIPTION
**Pull request by marco-theraforge**
Hope this helps, 

Marco

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -580,9 +580,9 @@ final class AnimatedView: UIView, AnimatedViewProtocol {
 }
 ```
 
-Types implementing design patterns are usually named with the pattern name at the end (e.g. `ViewBuilder`, `DisplayingStrategy`).
+Types implementing design patterns are usually named with the pattern name at the end (e.g., `ViewBuilder`, `DisplayingStrategy`).
 
-No prefixes are used (e.g., just `PriceCalculator` instead of `XYZPriceCalculator`), since there's no necessity for that in Swift (unlike for example, Objective-C).
+No prefixes may be used (e.g., just `PriceCalculator` instead of `XYZPriceCalculator`), since there's no necessity for that in Swift (othen than to maintain consistency with Objective-C libraries and components).
```
Comment (lazarevzubov) [inline README.md]: It seems there's a typo: `othen` instead of `other`.

**Comment by lazarevzubov** [comment]
Hi Marco,

Thank you for this great job and making this document undoubtedly better! I only would be happy to see a typo fixed before merge.

Best,
Nikita

**Comment by marco-theraforge** [comment]
Hi Nikita,

I am a GitHub newbie so I still need to get acquainted with it.

I have fixed the “othen” typo in my forked repo. The pull request seems to reflect that.

Please let me know if I need to do anything else...

Thanks,

Marco

**Comment by lazarevzubov** [comment]
Thanks a lot!

**Pull request merged**